device/mellanox/x86_64-mlnx_msn4600c-r0/Mellanox-SN4600C-D100C12S2/hwsku.json

```diff
@@ -0,0 +1,347 @@
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet90": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet92": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet94": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet96": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet98": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet100": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet102": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet104": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet106": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet108": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet110": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet112": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet114": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet116": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet118": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet120": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet122": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet124": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet126": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet128": {
+            "default_brkout_mode": "1x100G[50G,40G,25G,10G,1G]"
+        },
+        "Ethernet132": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet134": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet136": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet138": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet140": {
+            "default_brkout_mode": "1x100G[50G,40G,25G,10G,1G]"
+        },
+        "Ethernet144": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet146": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet148": {
+            "default_brkout_mode": "1x10G[100G,50G,40G,25G,1G]"
+        },
+        "Ethernet152": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet154": {
+            "default_brkout_mode": "2x50G[25G,10G,1G]"
+        },
+        "Ethernet156": {
+            "default_brkout_mode": "1x10G[100G,50G,40G,25G,1G]"
```

Comment:
Shouldn't this mode be updated in platform.json too?

Reply:
Yes. I hadn't included this in the PR but I will.